src/mds/MDCache.cc

```diff
@@ -448,13 +448,6 @@ void MDCache::create_mydir_hierarchy(MDSGather *gather)
     stray->store_backtrace(gather->new_sub());
   }
 
-  CInode *journal = create_system_inode(MDS_INO_LOG_OFFSET + mds->whoami, S_IFREG);
-  string name = "journal";
-  CDentry *jdn = mydir->add_primary_dentry(name, journal);
-  jdn->_mark_dirty(mds->mdlog->get_current_segment());
-
-  mydir->fnode.fragstat.nfiles++;
-  mydir->fnode.rstat.rfiles++;
```

Comment:
This bit makes me more nervous than the earlier patches. I think we hardcode some assumptions about the value of these nfiles/rfiles counters in a few places; did you check for any issues with that?